### PR TITLE
Change by

### PR DESCRIPTION
**Pull request by kimquy**
Implement this feature https://github.com/antonmi/espec/issues/181

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -569,6 +569,7 @@ Test if call of function1 change the function2 returned value to smth or from to
 expect function1 |> to(change function2)
 expect function1 |> to(change function2, to)
 expect function1 |> to(change function2, from, to)
+expect function1 |> to(change function2, [by: value])
```
Comment (antonmi) [inline README.md]: Seems like we needn't square brackets here.
Just: `to(change function2, by: value)`.
It's prettier, I think.

File: spec/assertions/change_spec.exs [hunk 2]
```diff
@@ -26,6 +26,11 @@ defmodule ChangeSpec do
       message = expect(f1()).to change(f2(), 0, 1)
       expect(message) |> to(end_with "changes the value of `&ChangeSpec.count/0` from `0` to `1`.")
     end
+
+    it "checks success with `by`" do
+      message = expect(f1()).to change(f2(), [by: 1])
```
Comment (antonmi) [inline spec/assertions/change_spec.exs]: And here we may remove []

**Comment by kimquy** [comment]
Hi @antonmi I updated the PR. Thanks for the feedbacks.

**Comment by antonmi** [comment]
Thank @kimquy for the collaboration!

**Pull request merged**